python/operations.cpp

```diff
@@ -133,4 +134,18 @@ void init_operations(py::module &m) {
 
   bind_contains_events<Variable>(m);
   bind_contains_events<DataArray>(m);
+
```

Comment:
Description is probably enough, but could have been put in a "decompose" submodule.